tests/Oryx.Integration.Tests/PythonEndToEndTests.cs

```diff
@@ -272,6 +272,24 @@ public async Task CanBuildAndRunPythonApp_UsingPython37()
                 });
         }
 
+        [Fact(Skip = "AB#896178")]
+        public async Task CanBuildAndRunPythonApp_WithBuildpack()
```

Comment:
though php integration tests are fastest today, but it will be nice to follow the same pattern as mentioned above, i.e. move these buildpack tests under a new class